Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/LineToolIntegrationTest.java

```diff
@@ -218,10 +232,13 @@ public void testCheckmarkLineFeatureChangeShape() {
 
 		onDrawingSurfaceView()
 				.checkPixelColor(Color.BLACK, BitmapLocationProvider.MIDDLE);
+		IdlingRegistry.getInstance().unregister(idlingResource);
 	}
 
 	@Test
 	public void testCheckmarkLineFeatureChangeStrokeWidth() {
+		CountingIdlingResource idlingResource = launchActivityRule.getActivity().getIdlingResource();
+		IdlingRegistry.getInstance().register(idlingResource);
```

Comment:
You already invoked .register() in the setup method, so I think there is no need to additionally .register() at the start of the test.